### ISSUE TITLE
[Request] can we support autocomplete to filter list/

### ISSUE THREAD

**Issue by BennieMeng** [bombshell-dev/clack#203]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by natemoo-re** [comment]
Yes, I'm working on an autocomplete/filter list component currently!

**Comment by xiapi233** [comment]
Looking forward to this feature.

**Comment by avil13** [comment]
Thank you for your work.
Very much missing the filtering of the list to select the desired item.

**Comment by natemoo-re** [comment]
Closing — autocomplete filtering was added in [@clack/core@1.0.0-alpha.0](https://github.com/bombshell-dev/clack/releases/tag/%40clack%2Fcore%401.0.0-alpha.0) with the new `AutocompletePrompt`, and custom filter support was added in [@clack/prompts@1.0.0-alpha.10](https://github.com/bombshell-dev/clack/releases/tag/%40clack%2Fprompts%401.0.0-alpha.10).
